### PR TITLE
New installer

### PR DESCRIPTION
**Pull request by howardjohn**
Prometheus scraping is still broken, everything else works (load + stability)

### CODE REVIEW

File: perf/load/common.sh [hunk 1]
```diff
@@ -16,6 +16,7 @@ function run_test() {
 
    # remove stdio rules
   kubectl --namespace istio-system delete rules stdio stdiotcp || true
+  kubectl --namespace istio-ingress delete rules stdio stdiotcp || true
```
Comment (mandarjog) [inline perf/load/common.sh]: This and the previous line is no longer needed. We don’t create stdio by default now

**Comment by howardjohn** [comment]
@mandarjog @costinm I have this running at http://grafana.howardjohn.qualistio.org. It only works with master, not 1.1 because of the injector (https://github.com/istio/installer/issues/65). Still need to make changes to update prometheus and grafana charts in installer for it to really work, I have those locally but not merged yet

File: perf/istio-install/values-new-installer.yaml [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+grafana:
+  enabled: true
+  prometheusNamespace: istio-prometheus
+  # datasources:
+  #   datasources.yaml:
+  #     apiVersion: 1
+  #     datasources:
+  #       - name: Prometheus
+  #         type: prometheus
```
Comment (mandarjog) [inline perf/istio-install/values-new-installer.yaml]: how come this is commented out?

Comment (howardjohn) [reply perf/istio-install/values-new-installer.yaml]: I changed this to use the built in support for prometheus operator that Doug added to the new installer, so now we can use the default datasource that comes with the grafana install.

**Comment by mandarjog** [comment]
Fails with `docker: command not found` during build

**Comment by howardjohn** [comment]
@mandarjog just had to rebase. This is ready to go on this side -- some more changes on the installer needed to make everything work right (some metrics still don't work)

**Comment by costinm** [comment]
To be clear: in the end, we need 1.1 running in istio-system, and at least 2 'environments' with the new installer ( for the tic-toc Mandar mentioned )

/lgtm

File: perf/istio-install/values-new-installer.yaml (hunk 2 above)
Comment (JimmyCYJ) [reply perf/istio-install/values-new-installer.yaml]: I cannot deploy Prometheus by running ./setup_istio_release.sh, could this be the reason?

**Pull request merged**